test/.env

```
@@ -0,0 +1,137 @@
+CADDY_TALAWA_API_DOMAIN_NAME=localhost
+CADDY_TALAWA_API_EMAIL=talawa@email.com
+CADDY_TALAWA_API_HOST=api
+CADDY_TALAWA_API_PORT=4000
+
+########## docker compose `cloudbeaver` container service ##########
+
+CLOUDBEAVER_ADMIN_NAME=talawa
+CLOUDBEAVER_ADMIN_PASSWORD=password
+CLOUDBEAVER_MAPPED_HOST_IP=127.0.0.1
+CLOUDBEAVER_MAPPED_PORT=8978
+CLOUDBEAVER_SERVER_NAME=Talawa CloudBeaver Server
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)